prime-dotnet-webapi/Configuration/Database/Seeded Tables/RemoteAccessTypeLicenseConfiguration.cs

```diff
@@ -0,0 +1,84 @@
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 2 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 3 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 4 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 5 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 6 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 8 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 9 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 10 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 12 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 13 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 14 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 15 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 17 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 18 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 19 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 47 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 51 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 59 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 66 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 67 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 87 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 88 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 89 },
+
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.FNHA, LicenseCode = 35 },
```

Comment:
Add comment "FNHA for additional nurse license classes"?

Reply:
Comment added.